l5kit/l5kit/environment/gym_metric_set.py

```diff
@@ -23,3 +23,50 @@ def build_metrics(self) -> List[metrics.SupportsMetricCompute]:
             metrics.DisplacementErrorL2Metric(),
             metrics.YawErrorCAMetric()
         ]
+
+
+class DisplacementCollisionMetricSet(metric_set.L5MetricSet):
```

Comment:
The name of this class is a little confusing, it says DisplacementCollisionMetricSet but we also have distance to ref traj here. I would just change it to be something like CLEMetricSet.